packages/drivers/odsp-driver/src/odspError.ts

```diff
@@ -9,6 +9,17 @@ import { parseAuthErrorClaims } from "./parseAuthErrorClaims";
 
 const nullToUndefined = (a: string | null) => a ?? undefined;
 
+function numberFromHeader(header: string | null): number | undefined {
+    if (header === null) {
+        return undefined;
+    }
+    const n = Number(header);
+    if (Number.isNaN(n)) {
```

Comment:
Maybe you can reverse the conditions and save 1 return statement?

Reply:
Reversing will not make a differences I believe (unless you mean remove return undefined and rely on it being default value - I do not like that, it makes code less readable, I hate that feature of TS :)).
I can inline, i.e.
> return Number.isNaN(n) ? undefined : n;

I think it's good as is :)